source/reference/minio-server/minio-server.rst

```diff
@@ -170,20 +170,97 @@ The command accepts the following arguments:
    :optional:
 
    Enable and configure a SSH File Transfer Protocol (``SFTP``) server.
-   Use multiple times to specify an address port and the path to the ssh private key to use as key-value pairs.
+   Use multiple times to specify each desired key-value pair.
 
-   Valid keys:
+   The following table lists valid keys.
+
+   .. list-table::
+      :header-rows: 1
+      :widths: 30 30 40
+      :width: 100%
+   
+      * - Key
+        - Description
+        - Valid values
+   
+      * - ``address``
+        - Port to use for connecting to SFTP.
+        - Any valid port number, typically ``8022``.
+   
+      * - ``ssh-private-key``
+        - Path to the user's private key file.
+        - Absolute path or relative path from current location to the key file to use.
+ 
+      * - ``pub-key-algos``
+        - Comma-separated list of the public key algorithms to support.
+        - 
+          .. code-block:: text
```

Comment:
Nit - code blocks in tables makes for severely crushed text.

At the same time monospacing each line is also messy.

I'd suggest one of two paths:

1. keep it plaintext and not worry about it
2. Collapse the second and third columns

Either way LGTM

Reply:
If it gets too problematic, we can circle back and fix it in a later pass. I'm just going to leave it like it is for now.